staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/compilation.go

```diff
@@ -65,6 +66,10 @@ type CompilationResult struct {
 	// MessageExpressionMaxCost represents the worst-case cost of the compiled MessageExpression in terms of CEL's cost units,
 	// as used by cel.EstimateCost.
 	MessageExpressionMaxCost uint64
+	// NormalizedRuleFieldPath represents the relative fieldPath specified by user after normalization.
+	NormalizedRuleFieldPath *field.Path
+	// FieldPathError represents the error while validating fieldPath specified.
+	FieldPathError *apiservercel.Error
```

Comment:
I don't think we need to store this, if we somehow get a CRD with a field path we're not happy with, we should just return errors without a custom field path